activation/post.go

```diff
@@ -89,6 +89,11 @@ func (c *PostClient) Execute(challenge []byte) (*types.PostProof, error) {
 }
 
 func (c *PostClient) Reset() error {
+	ok, err := c.IsInitialized()
+	if !ok || err != nil {
```

Comment:
This won't allow to reset when in `Crashed` state. 
You don't need to perform this validation at all, because `Initializer.Reset()` is already doing it by calling `Initializer.VerifyStarted()`.

Reply:
what happens if the process has started? can I reset while the process is creating files?

Reply:
If it started but not finished then it will be detected as `Crashed`. The in-progress state is supported only via the post RPC server (see [here](https://github.com/spacemeshos/post/blob/develop/rpc/api/api.proto#L126)). Same thing can be implemented in `PostClient`. It wasn't applicable to be implemented on the package itself because multiple clients can run it on the same datadir, while detecting it based solely on the filesystem is too complicated.

Reply:
You need to protect against the in-progress state. If not, you might reset during the initialization. If instead you'll check that initialization completed, then you won't be able to reset after a crash.

Reply:
ok so what is your recommendation? reset anyway?

Reply:
> You need to protect against the in-progress state.

In the same way as implemented in the post RPC server.